examples/simple/Error_Codes.csv

```diff
@@ -169,4 +169,4 @@ L-STDLIBE3,"Realloc overflows memory bound, returning null pointer.",,Possible u
 L-CEIE1,Memory malloced but never freed.,,Possible unintended behavior,X
 L-CCVE1,unsigned integer overflow.,,Possible unintended behavior,X
 L-CCVE2,conversion to unsigned integer outside the range that can be represented.,,Possible unintended behavior,X
-L-EXC3,Comparing a function pointer to a null pointer constant.,,Possible unintended behavior,X
+L-EXC3,Comparing a function pointer to a null pointer constant.,,Possible unintended behavior,X
```

Comment:
can you fix this please? noeol is generally bad for most purposes with text files.